addons/nuxeo-ai-core/elements/admin-ai-insight.js

```diff
@@ -0,0 +1,92 @@
+/**
+ @license
+ (C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class';
+import { I18nBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-i18n-behavior.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+
+/**
+ `admin-ai-insight`
+ @group Nuxeo UI
+ @element admin-ai-insight
+ */
+class AdminAiInsight extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+  static get is() {
+    return 'admin-ai-insight';
+  }
+
+  static get properties() {
+    return {
+      link: {
+        type: String,
+        value: '',
+      },
+    };
+  }
+
+  ready() {
+    super.ready();
+    this.$.aiInsight
+      .execute()
+      .then(({ url, token, projectId, urlCore }) => {
+        this.link = `${url}?token=${token}&projectid=${projectId}&urlCore=${urlCore}`;
+      })
+      .catch(() => {
+        this.fire('notify', { message: this.i18n('admin.ai.cloud.error') });
+      });
+  }
+
+  static get template() {
+    return html`
+      <style include="nuxeo-styles">
+        .flex {
+          @apply --layout-flex;
+        }
+        .page-insight {
+          background: --nuxeo-page-background;
```

Comment:
Just in case the variable is not defined we should defined a fallback
```suggestion
          background: var(--nuxeo-page-background, #f5f5f5);
```